src/User.js

```diff
@@ -1,14 +1,47 @@
 class User {
-  constructor(user){
+  constructor(user) {
     this.id = user.id;
     this.name = user.name;
-    this.customer = user.customer;
     this.userBookings = [];
+    this.bookingCost = 0;
+
+  }
+  // Methods I think I'll need
+
+  retrieveRoomBookings(bookingData) {
+
+    bookingData.filter(booking => {
+      if (this.id === booking.userID) {
+        this.userBookings.push(booking);
+      }
+    });
+    // console.log(this.userBookings)
+    return this.userBookings;
```

Comment:
Since you are pushing in the entire booking, an alternative way to accomplish this is to let `.filter` do it's thing, so...

```
const filteredBookings = bookingData.filter...
```

And then at the end (so line 19 here), just set `this.userBookings` to `filteredBookings`. 

How you are doing this works, but I would tend to push individual elements only if you're not pushing the entire element (like we did today with `item.item` or the hand-crafted object). So here, since `filter` keeps the entire element in the filtered array, it's accomplishing the same thing ☺️